flang/test/Fir/box-offset-codegen.fir

```diff
@@ -0,0 +1,44 @@
+// Test fir.box_offset parse/print/parse/print identity.
+// RUN: fir-opt %s | fir-opt | FileCheck %s
```

Comment:
I guess you meant this to be a codegen and not parse/print test.

Reply:
Absolutely, thanks for the review! I updated the test with the actual codegen test.